apps/meteor/client/views/omnichannel/directory/OmnichannelDirectoryPage.tsx

```diff
@@ -30,7 +31,7 @@ const OmnichannelDirectoryPage = (): ReactElement => {
 
 	const handleTabClick = useCallback((tab) => (): void => directoryRoute.push({ tab }), [directoryRoute]);
 
-	const [chatReload, setChatReload] = useState();
+	const chatReload = () => queryClient.invalidateQueries({ queryKey: ['current-chats'] });
```

Comment:
@MartinSchoeler , seems the refactor associated with this changed the "reactivity" of the view.

Previously, it was getting "refetched" everytime the page opened, now, it's getting stale until refresh, so we need a way to know that this query needs to be invalidated

Reply:
Noticed that the `staleTime` is set to infinity which would be the reason y you had to refresh the whole page to see the latest value. I believe we need to change it to something reasonable like 10 seconds. WDYT?
https://github.com/RocketChat/Rocket.Chat/blob/de67df97dc15666b9db8627bf7bb514e6032cb83/apps/meteor/client/views/omnichannel/currentChats/hooks/useCurrentChats.ts#L13

Reply:
I think that works. We probably need a stream that let these queries now when to refresh.
One that gets triggered when a room is taken/closed/updated etc so these areas that interact with rooms here may refetch.

Do you know if we have something similar? Otherwise, reducing stale time should work 👀 for now